src/Core/src/Platform/Gtk/MauiDatePicker.cs

```diff
@@ -1,35 +1,443 @@
 ﻿using System;
 using System.Runtime.Serialization;
+using System.Linq;
+using Gtk;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Graphics.Platform.Gtk;
 
 namespace Microsoft.Maui.Platform
 {
-	public class MauiDatePicker : Gtk.Label
+#pragma warning disable 612
```

Comment:
@parhamsaremi what's the reason for this?

Reply:
If I remove that we'll get these errors. https://gist.github.com/parhamsaremi/fa6806f4c2f69b4a609f7837706a044a

I'll explain in a comment why this is added :thinking:

Reply:
then use the pragma only in the places where these warning happens, not in the whole file (and still put comments in each explaining why)

Reply:
sure :+1: